### ISSUE TITLE
InterstitialScreen: accessibility testing issue set 1

### ISSUE THREAD

**Issue by amal-k-joy** [carbon-design-system/ibm-products#7458]
These are the set 1 of issues raised by Raghu on his accessibility testing.

[Carbon for IBM Components_Interstitialscreen_12thMay_Updated.xlsx](https://github.com/user-attachments/files/20186087/Carbon.for.IBM.Components_Interstitialscreen_12thMay_Updated.xlsx)

Issue   title/Summary | Issue Description | Recommended Fix | User Impact | Developer comments
-- | -- | -- | -- | --
reading   order of Close Button is not  following   the visual order  of the Element. | On Arrow key navigation  , "Close Button " announced after   the H2  and H3 level headings  (which mentioned below) , Here  Screen reader  reading order is not matching with Logical   order . | Recommend to  make this "Close Button "   announced  as the First Element in the   Modal . | If the components are  not announced in a  Meaningful order /In an order How its   visually placed in the  screen - its   may distract its   meaning , How it's   suppose to be understood, impact    screen reader users. | This is a known behaviour of   carbon compose modal. Ref   [18262](https://github.com/carbon-design-system/carbon/issues/18262). Close button is placed after   the heading elements.
Screen   reader is announcing the  Elements   which is not visually available in the screen | Screen reader is announcing   the  Elements which is not visually   available in the screen - creating confusion    for Screen reader users.     Note : This element is not  keyboard   operable /Tab key reachable . | Screen reader is not expected to   announce the  Elements which is not   visually available in the screen :     Make sure  the visually hidden   elements are hidden from screen reader detection as well . | If the screen reader identifies   the visually hidden content, it tends to mislead users who are blind. | Behaviour of compose modal to trap focus via sentinal elements. Fixed by using the carbon _Feature flag_ (`enableExperimentalFocusWrapWithoutSentinels`)
Heading   level hierarchy  is not maintained        1. H1 Placed after H2 and H3 Level Headings .     2. H1  is not assigned  for the Main Headline  of the Modal. | Heading level hierarchy  is not maintained- Screen Reder  users may struggle to identify the content   structure of the page . | For the benefits of Screen Reder   users :     1. Maintain heading levels    hierarchy       2. H1 for the Main Header for the Modal    at the Top , H2 for the Sub Headlines of H1, H3 for the subhead lines   of H2  and so on | If page headings are not in the   right sequence, screen reader users find it difficult to comprehend the   hierarchical structure of the page content. | Fixed. Title and subTitle changed   to h1 and h2
